tests/gnrc_udp/Makefile

```diff
@@ -28,6 +29,8 @@ USEMODULE += ps
 USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
 
+CFLAGS += -DGNRC_PKTBUF_SIZE=8192
```

Comment:
I noticed from running the release spec for tasks 4 and 7 that the example guides for those tasks specify use of this test, `tests/gnrc_udp`. Are you concerned with setting this pktbuf size for all of those tests?

This question really is part of both this PR and the release specs, but the larger question is whether we want to change the default pktbuf size for this test.

Reply:
> I noticed from running the release spec for tasks 4 and 7 that the example guides for those tasks specify use of this test, `tests/gnrc_udp`. Are you concerned with setting this pktbuf size for all of those tests?

Simple answer: No. The nodes the tests are run on shoul dbe able to run fit those extra 2KB of RAM. Some boards might need to be added to BOARD_INSUFFICIENT_MEMORY, but there already some in there. I am more concerned about people reporting the test to fail because they forgot to set the `GNRC_PKTBUF_SIZE` or wasting their time until realizing they forgot it.